src/test/scala/nl/vroste/zio/kinesis/client/DynamicConsumerTest.scala

```diff
@@ -6,64 +6,82 @@ import nl.vroste.zio.kinesis.client.Client.ProducerRecord
 import nl.vroste.zio.kinesis.client.serde.Serde
 import software.amazon.kinesis.exceptions.ShutdownException
 import zio._
+import zio.blocking.Blocking
 import zio.clock.Clock
 import zio.console._
 import zio.duration._
 import zio.stream.{ ZStream, ZTransducer }
-import zio.test.Assertion._
 import zio.test.TestAspect._
 import zio.test._
 
 object DynamicConsumerTest extends DefaultRunnableSpec {
   import TestUtil._
 
+  private val createStreamLayers = LocalStackLayers.kinesisAsyncClientLayer >>> AdminClientLive.layer
```

Comment:
You can provide layers to the entire test suite, just once.

Reply:
I overlooked that - good idea - I'll give it a go

Reply:
So ATM each test has some subtle variation in layer requirements that I think are not possible to satisfy in one go at the suite level. A consolidation may be be possible but would require a pretty big refactor and effort. What do you think?